WhaleSpotting/Services/SightingsService.cs

```diff
@@ -53,6 +53,7 @@ public async Task<List<SightingResponseModel>> SearchSighting(SearchSightingRequ
                 .Where(s => searchSighting.SightedTo == null || s.SightedAt <= searchSighting.SightedTo)
                 .Where(s => searchSighting.OrcaPod == null || s.OrcaPod == searchSighting.OrcaPod)
```

Comment:
Just realise, could you make all the string search fields 
```suggestion
                .Where(s => string.IsNullOrEmpty(searchSighting.OrcaPod) || s.OrcaPod == searchSighting.OrcaPod)
```

Reply:
done